### PR TITLE
fix: [UPC-5820] tvOS app cant be built with Google WebRTC lib

### PR DESCRIPTION
**Pull request by AtinYehUI**
[ticket](https://ubiquiti.atlassian.net/browse/UPC-5820)
Screenshot 2023-08-24 at 7 04 08 PM

### CODE REVIEW

**Comment by ogsts** [comment]
@AtinYehUI Jenkins build does not include TV OS ?

It finished successfully

https://pdx-build-aerith.rad.ubnt.com/blue/organizations/jenkins/unifi-protect%2FEMS%2FGoogleWebRTC/detail/ubnt_m116/2/pipeline

File: sdk/BUILD.gn [hunk 1]
```diff
@@ -677,13 +686,11 @@ if (is_ios || is_mac) {
...
-          sources += [
-              "objc/components/capturer/RTCCameraVideoCapturer.h",
-              "objc/components/capturer/RTCCameraVideoCapturer.m",
-          ]
-        }
+      if (ios_device_name != "appletv") {
+        sources += [
+            "objc/components/capturer/RTCCameraVideoCapturer.h",
+            "objc/components/capturer/RTCCameraVideoCapturer.m",
+        ]
```
Comment (ogsts) [inline sdk/BUILD.gn:693]: # UI Customization End 
?

**Comment by UI-RayanWang** [comment]
Should we release new version based on M116? Looks like this PR will merge changes into M113.

File: sdk/BUILD.gn (hunk 1 above)
Comment (AtinYehUI) [reply sdk/BUILD.gn:693]: Tanks. lost # UI Customization Begin

**Comment by AtinYehUI** [comment]
> @AtinYehUI Jenkins build does not include TV OS ?
> 
> It finished successfully
> 
> https://pdx-build-aerith.rad.ubnt.com/blue/organizations/jenkins/unifi-protect%2FEMS%2FGoogleWebRTC/detail/ubnt_m116/2/pipeline

iOS build script 
Always build all apple platform .iPhone, tvOS, Simulate,  maccatalyst

**Comment by ogsts** [comment]
> Always build all apple platform .iPhone, tvOS, Simulate, maccatalyst

@UI-RayanWang What did you mean with - cant be built for TV OS ?
Jenkins build successfuly built it.

So, it can be built ? or not ?

**Comment by UI-RayanWang** [comment]
> > Always build all apple platform .iPhone, tvOS, Simulate, maccatalyst
> 
> @UI-RayanWang What did you mean with - cant be built for TV OS ? Jenkins build successfuly built it.
> 
> So, it can be built ? or not ?

@ogsts It can be built, however the issue is the framework can't be include/link from a tvOS project.

**Comment by AtinYehUI** [comment]
This PR is to correct the m113 version. After cherrypick to m116.

M116 have the same header not remove header.
Because header objc class api not support appletv platform. but build.gn no remove copy header.

**Review by ogsts — APPROVED** [review]

**Comment by ogsts** [comment]
yes, need to be rebased to m116 first

**Review by UI-RayanWang — APPROVED** [review]

**Pull request merged**